Makefile

```diff
@@ -59,14 +59,19 @@ build-e2e:
 
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
+	# Run Disruptive tests first. Those might include things causing all masters to restart, and should run serially
+
+	# TODO: Disabled, until all disruptive tests are prefixed with post-conditions, allowing cluster to recover
+	# hack/ci-integration.sh $(GINKGO_ARGS) -focus="\[Disruptive\]" || (hack/junitmerge.sh && exit 1)
```

Comment:
Does this currently run any tests? Is it worth merging it uncommented and just having it be a no-op for now?

Reply:
I think we can uncomment it, but revert to the state of https://github.com/openshift/cluster-api-actuator-pkg/commit/d8f139d9f67dfd10dff4b7cfe3ea60c9cefba429 and https://github.com/openshift/cluster-api-actuator-pkg/commit/e3a1a4fb39a61566087d97d2139cadebf9482c87 first. Let'd mark proxy tests as disruptive for now, and consider adding post/pre conditions later @JoelSpeed

Reply:
I don't really follow what you mean by reverting? As far as I can tell, this should be good to merge if we just uncomment this line. What are you concerned about in particular?

Reply:
No, just trying to recall conversation in slack on this topic. SGTM and updated, PTAL @JoelSpeed